app/helpers/notifier_helper.rb

```diff
@@ -2,4 +2,13 @@ module NotifierHelper
   def fp(text)
     format_paragraph(text, 72, 0)
   end
+
+  def link_to_user(display_name)
+    link_to(
+      display_name,
+      user_url(display_name, :host => SERVER_URL),
+      :target => "_blank",
+      :style => "text-decoration: none; color: #222; font-weight: bold"
+    )
+  end
 end
```

Comment:
I added the `link_to_user` helper to declutter the template a bit, but then it means this Ruby file now contains some CSS instructions. What's the proper Rails way to do this?